feedback/filters.py

```diff
@@ -193,7 +193,7 @@ class FeedbackFilter(django_filters.FilterSet):
                                           widget=forms.CheckboxSelectMultiple())
     flags = FlagFilter(label=_("Flags"))
     tags = ColortagIEAndOrFilter(queryset=FeedbackTag.objects.none(), label=_("Tags"))
-    student_tags = ColortagIEAndOrFilter(queryset=StudentTag.objects.none(), label=_("Student tags"))
+    student_tags = ColortagIEAndOrFilter(queryset=StudentTag.objects.none(), field_name='student__tags', label=_("Student tags"))
```

Comment:
This is already in master? https://github.com/Aalto-LeTech/mooc-jutut/commit/678e70268f77026feb26e05b935810f9a3f0b263